File: src/java/magma/app/Whitespace.java
package magma.app;

import magma.api.option.None;
import magma.api.option.Option;

record Whitespace() implements Parameter {
    @Override
    public String generate() {
        return "";
    }

    @Override
    public Option<Definition> asDefinition() {
        return new None<Definition>();
    }
}

File: src/node/magma/app/Whitespace.ts
import { Parameter } from "../../magma/app/Parameter";
import { Definition } from "../../magma/app/Definition";
import { Option } from "../../magma/api/option/Option";
import { None } from "../../magma/api/option/None";
class Whitespace implements Parameter {
	generate(): string {
		return "";
	}
	asDefinition(): Option<Definition> {
		return new None<Definition>();
	}
}
